tests/roles/glance_adoption/tasks/check_glance.yaml

```diff
@@ -4,19 +4,7 @@
       ansible.builtin.shell: |
         {{ shell_header }}
         {{ oc_header }}
-
-        STATUS=$(oc get pod --selector=service=glance -o jsonpath='{.items[*].status.phase}{"\n"}');
-        code=1
-        IFS=" " read -r -a STATUS <<< "$STATUS"
-        for i in "${STATUS[@]}"; do
-            if echo "$i" | grep -v Running; then
-                # if at least one instance is not Running, return
-                exit $code
-            else
-                code=0
-            fi
-        done
-        exit $code
+        oc wait pod --for condition=Ready --selector=service=glance
```

Comment:
I'm ok with this change as long as CI is happy with that. I think the main point here was about wait for all the Glance pods to reach `Running` phase, but this looks better. Thanks Manoj!

Reply:
Yeah, wait for pod condition 'Ready' is simplest way to ensure that all the containers of the pod are in 'Running' phase and pod is able to serve requests.